src/vaults/locked/StrategyVaultV2.sol

```diff
@@ -22,6 +22,39 @@ contract StrategyVaultV2 is StrategyVault, NftGate, HarvestStorage {
         emit Deposit(caller, receiver, assets, shares);
     }
 
+        function _withdraw(address caller, address receiver, address owner, uint256 assets, uint256 shares)
+        internal
+        virtual
+        override
+    {
+        // If vault is illiquid, lock shares
+        if (!epochEnded) {
+            uint govShares = _getWithdrawalFee(shares, owner);
```

Comment:
Debt Escrow resolve debt through calling `vault.redeem`. So we don't need to withdraw fees now.

Reply:
this will incur double fees from user.